mod_reforged/hooks/config/strings.nut

```diff
@@ -1541,7 +1548,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 			{
 				Type = ::UPD.EffectType.Active,
 				Description = [
-					"Gain the [Encourage|Skill+rf_encourage_skill] skill that allows you to increase the [Morale|Concept.Morale] of an ally."
+					"Gain the [Encourage|Skill+rf_encourage_skill] skill that allows you to increase the [morale|Concept.Morale] of an ally."
```

Comment:
Minor: We're going with `which allows` with pretty much every other skill (and that's how vanilla does it in perk tooltips too) so let's change this from `that allows` to `which allows`.